src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -267,6 +269,10 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
                 tabControl.setTab(tabControl.TABS.DIRECTIONS);
             }
         }
+
+        if (key === 'origin') {
```

Comment:
The `onTypeahead` functions all run whenever the typeahead sends its signals, so components can can keep their data in sync.  I.e. directions and explore are already updating their local variables from the events, then deciding whether to do anything more based on whether they're active.  So adding this to Explore's `onTypeahead` methods inside the `if (key === 'origin')` (which is currently missing from `onTypeaheadSelected`, which is a bug) but outside the tab check would work fine and would make more sense to me.